vars/rollingUpgradePipeline.groovy

```diff
@@ -42,52 +42,127 @@ def call(Map pipelineParams) {
-                                                echo "end test ....."
-                                                """
+                                        stage("Upgrade from ${base_version}") {
+                                            wrap([$class: 'BuildUser']) {
+                                                dir('scylla-cluster-tests') {
+                                                    checkout scm
+                                                    try {
+                                                        sh """
+                                                        #!/bin/bash
+                                                        set -xe
+                                                        env
+                                                        export SCT_NEW_CONFIG=yes
+                                                        export SCT_CLUSTER_BACKEND=gce
+
+                                                        export SCT_CONFIG_FILES=${pipelineParams.test_config}
+                                                        export SCT_SCYLLA_VERSION=${base_version}
+                                                        export SCT_NEW_SCYLLA_REPO=${pipelineParams.params.new_scylla_repo}
+
+                                                        export SCT_POST_BEHAVIOR_DB_NODES="${params.post_behavior_db_nodes}"
+                                                        export SCT_POST_BEHAVIOR_LOADER_NODES="${params.post_behavior_loader_nodes}"
+                                                        export SCT_POST_BEHAVIOR_MONITOR_NODES="${params.post_behavior_monitor_nodes}"
+                                                        export SCT_INSTANCE_PROVISION=${pipelineParams.params.get('provision_type', '')}
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+
+                                                        export SCT_GCE_IMAGE_DB=${pipelineParams.gce_image_db}
+                                                        export SCT_SCYLLA_LINUX_DISTRO=${pipelineParams.linux_distro}
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC="\$SCT_AMI_ID_DB_SCYLLA_DESC-\$SCT_SCYLLA_LINUX_DISTRO"
+
+                                                        export SCT_WORKAROUND_KERNEL_BUG_FOR_IOTUNE=${pipelineParams.workaround_kernel_bug_for_iotune}
+
+                                                        echo "start test ......."
+                                                        ./docker/env/hydra.sh run-test ${pipelineParams.test_name} --backend ${params.backend}  --logdir /sct
```

Comment:
This PR added collect_logs stage, then we don't want logs are collected by this stage.

In vars/longevityPipeline.groovy, disable collect_logs for run-test stage. It seems we also need to disable collect_logs for above stage to avoid duplicated log collection.

Reply:
https://github.com/scylladb/scylla-cluster-tests/blob/master/vars/longevityPipeline.groovy#L90

```
                export SCT_COLLECT_LOGS=false
                .......
                echo "start test ......."
                ./docker/env/hydra.sh run-test ${pipelineParams.test_name} --backend ${params.backend}  --logdir /sct
                 echo "end test ....."
```

Reply:
better to disable it in the rolling upgrade YAML